templates/aws_s3_backup.sh.j2

```diff
@@ -33,7 +33,7 @@ function main {
 		ok "Backup finished!"
 	fi
 
-	backup_path="$( echo "${response}" | python -c"import sys,json;j=json.loads(sys.stdin.read());print j['path']" )"
+	backup_path="$( echo "${response}" | jq -r '.["path"]' )"
```

Comment:
There are a few double quotes within double quotes.  I assume you tested this and it work, but ot looks a bit iffy.

Reply:
Yep this works, I've tested it for reals